tests/src-android/cgeo/geocaching/storage/DataStoreTest.java

```diff
@@ -97,6 +97,66 @@ public static void testStoredLists() {
         }
     }
 
+    public static void testMoveToList() {
+
+        int listId1 = StoredList.STANDARD_LIST_ID;
```

Comment:
If there can be an exception in `DataStore.createList`, you will remove the wrong list. If there can be no exception, you can initialize `list1` and friends with their right value directly and make them `final`.